src/tsconfig.json

```diff
@@ -11,6 +11,8 @@
 			"@root/*": ["*"]
 		},
 		"composite": true,
+		"sourceMap": true,
+		"inlineSources": true,
```

Comment:
```suggestion
```

`sourceMap` is inherited and `inlineSources` should not be needed. Sentry's page is pretty outdated for as far as I can tell tbh.